RecoTracker/LST/interface/LSTOutput.h

```diff
@@ -4,6 +4,8 @@
 #include <memory>
 #include <vector>
 
+enum LSTTCType { T5=4, pT3=5, pT5=7, pLS=8 };
+
 class LSTOutput {
 public:
```

Comment:
```suggestion
class LSTOutput {
public:
  enum TCType { T5=4, pT3=5, pT5=7, pLS=8 };
```
to avoid placing in global scope.

or use scoped enum `enum class `